fideslog/api/routes/health.py

```diff
@@ -1,14 +1,12 @@
 from fastapi import APIRouter, Request, status
 from fastapi.responses import JSONResponse
 
-from ..config import config
-from ..endpoints.events import router as event_router
+from fideslog.api.config import config
 
-api_router = APIRouter()
-api_router.include_router(event_router)
+health_router = APIRouter()
```

Comment:
Now that there's a dedicated `health_router`, we can move the `tags` from the endpoint definition on L56 to this `APIRouter()` invocation.